src/Symfony/Component/Notifier/Bridge/ContactEveryone/README.md

```diff
@@ -19,6 +19,30 @@ This bridge uses the light version of Contact Everyone API.
 
 See your account info at https://contact-everyone.orange-business.com/#/login
 
+Adding Options to a Message
+---------------------------
+
+With a Contact everyone Message, you can use the `ContactEveryoneOptions` class to add
+message options.
+
+```php
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Bridge\ContactEveryone\ContactEveryoneOptions;
+
+$sms = new SmsMessage('+1411111111', 'My message');
+
+$options = (new ContactEveryoneOptions())
+    ->diffusionName('My label')
```

Comment:
the method should be changed to accept a string also (would be consistent with the constructor of the transport)
@franckranaivo where can I find the doc that tells about this parameter?

Reply:
Sorry, for the late answer. I picked these parameters (including diffusionName) from their manuals. But yes it should accept string. Here is the manual, idk if there is an english version.
https://ceo-be.multimediabs.com/attachments/hosted/lightApiManualsFR
They start to talk about these on page 9.

Apparently, it is useful for categorizing sms logs.